src/app.ts

```diff
@@ -13,14 +13,15 @@ import { autoUpdater } from 'electron-updater'
 import { is } from 'electron-util'
 import log from 'electron-log'
 import electronDl from 'electron-dl'
-import electronContextMenu from 'electron-context-menu'
 
-import config, { ConfigKey } from './config'
+import config, { ConfigKey, LastWindowState } from './config'
 import { init as initDebug } from './debug'
 import menu from './menu'
 import { init as initCustomStyles } from './custom-styles'
 import { platform, getUrlAccountId } from './helpers'
 
+import electronContextMenu = require('electron-context-menu')
```

Comment:
`import from`

Reply:
This doesn't work as they changed the export to commonjs.

https://github.com/sindresorhus/electron-context-menu/releases/tag/v0.12.0

Reply:
Interesting, thanks for sharing. Why would they change this to commonjs? Couldn't find any explanation.

Reply:
I assume that they changed it so it could be used by commonjs apps.

Reply:
I think using `import * as electronContextMenu from 'electron-context-menu'` should work too then?

Reply:
Yes, but it isn't the right solution according to this.

https://stackoverflow.com/a/35706271/5641674

Reply:
Thanks, we stick to this then 👍